### PR TITLE
Keep mutex alive to avoid race condition in openmutexpos4 test

### PR DESCRIPTION
**Pull request by wtgodbe**
This should fix the race condition in https://github.com/dotnet/coreclr/issues/5562

CC @sergiy-k @RussKeldorph

### CODE REVIEW

File: tests/src/baseservices/threading/mutex/openexisting/openmutexpos4.cs [hunk 1]
```diff
@@ -29,6 +29,7 @@ public void CreateMutexThread()
         mutex.ReleaseMutex();
 		
 		exitEvent.WaitOne();
```
Comment (RussKeldorph) [inline tests/src/baseservices/threading/mutex/openexisting/openmutexpos4.cs]: Could you fix the indentation on this line while you're in here?

**Comment by RussKeldorph** [comment]
I find the documentation for the third parameter (`createdNew`) on this page:
https://msdn.microsoft.com/en-us/library/bwe34f1k%28v=vs.110%29.aspx to be inconsistent and confusing, though http://stackoverflow.com/questions/11754921/mutex-constructor-in-net-is-strange attempts to sort it out.

As long as the remaining race between ReleaseMutex in T1 and the constructor in T2 has no bearing on the value in the third parameter (`createdNew`) after the constructor call, then LGTM.

**Comment by sergiy-k** [comment]
LGTM

File: tests/src/baseservices/threading/mutex/openexisting/openmutexpos4.cs (hunk 1 above)
Comment (RussKeldorph) [inline tests/src/baseservices/threading/mutex/openexisting/openmutexpos4.cs]: Looks like the problem is tabs.  I recommend untabifying the file while you're here, but feel free not to if you feel that should be done separately.

**Pull request merged**